java/src/org/openqa/selenium/support/events/EventFiringDecorator.java

```diff
@@ -297,8 +294,8 @@ private void fireAfterEvents(
 
   private String createEventMethodName(String prefix, String originalMethodName) {
     return prefix
-        + originalMethodName.substring(0, 1).toUpperCase()
-        + originalMethodName.substring(1);
```

Comment:
Please run the `format.sh` script to double check this changes. It is located under the `scripts` directory.